.github/ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,16 @@
-### Summary
-Tell us about your problem. What operating system are you using? How big was your dataset?
+## Summary
+What are you thinking?
```

Comment:
Maybe, "What's on your mind?" or "What are you thinking about?" There's a part of me that's reading that can read this in a more aggressive way if I was a first-timer, e.g. "WHAT are you thinking?!"